bindings/cpp/session.cpp

```diff
@@ -2287,6 +2341,110 @@ async_iterator_result session::cancel_iterator(const key &id, uint64_t iterator_
 	return iterator(id, data);
 }
 
+async_iterator_result session::server_send(const std::vector<std::string> &keys, uint64_t iflags, const std::vector<int> &groups)
+{
+	if (get_groups().empty()) {
+		async_iterator_result result(*this);
+		async_result_handler<iterator_result_entry> handler(result);
+		handler.complete(create_error(-ENXIO, "server_send: local group list is empty"));
+		return result;
+	}
+
+	if (groups.empty()) {
```

Comment:
Is it expected?

In some case it could be useful, for example, if you want to convert data locally, but it should be additionally described in the docs with details and examples. It isn't stop requirement for this pull request, but it is recommendation from the category "good to have".